### ISSUE TITLE
Ability to Sort All Apps by Date DESC of Latest Uploaded Build

### ISSUE THREAD

**Issue by zmugraby** [microsoft/appcenter#972]
**Describe the solution you'd like**
In HockeyApp, we had the ability to sort All Apps by latest uploaded build or update.  In AppCenter, we lost this capability.  Adding an additional sortable column showing release date will be super useful.

**Describe alternatives you've considered**
Sorting the all apps list on legacy Hockeyapp

**Additional context**
Appcenter with no Release Date Column:
<img width="1204" alt="Screen Shot 2019-09-04 at 9 10 13 AM" src="https://user-images.githubusercontent.com/8845313/64229907-d9534a80-cef3-11e9-81fb-2cb442e00bf4.png">

HockeyApp with Release Date Column:
<img width="1483" alt="Screen Shot 2019-09-04 at 9 10 22 AM" src="https://user-images.githubusercontent.com/8845313/64229911-dbb5a480-cef3-11e9-81bd-7ee72813003f.png">

**Comment by blparr** [comment]
Hello @zmugraby ! Thanks for your feature request. We'll be reviewing your request and prioritize accordingly. 

Thanks!

**Comment by SpaceApeGamesSy** [comment]
I'll +1 this, with a large number of apps, it's useful for teams to be able to sort them by recent update, to cut down time searching through lists. Yes I realise there's a search box. None of my teams ever use it :(

We've taken some initial steps to reduce the problem by using two separate orgs for live and archived applications, but it only goes so far and with 100+ builds still in the list, this would be super useful.

**Comment by smpeleato** [comment]
+1 for this as well. We really miss the ability to sort Apps by last updated like you could with HockeyApp.

**Comment by Brantone** [comment]
Definitely +1 on this.
Ability to sort is crucial.

**Comment by wivlaro** [comment]
Why is this issue closed? It isn't done. Especially now with the retiring of the mobile apps, this is as important as ever. Please reopen.

**Comment by wivlaro** [comment]
Is this issue going to be addressed? We still consider this as really important in our organisation where there are a great many apps with variations of them.

**Comment by wivlaro** [comment]
I still wish we had this and wish for it every time I go to download a build.

**Comment by Brantone** [comment]
(bump)

**Comment by Brantone** [comment]
Still useful.

**Comment by wivlaro** [comment]
No. I still miss this. We're even re-evaluating our use of appcenter these days and this is the kind of unaddressed issue that prompts these conversations

**Comment by Brantone** [comment]
How hard is it to add a different sort option?
Are there parts of AC that can be opened for contribution?

**Comment by wivlaro** [comment]
Nope, not stale. Still useful. 👊

**Comment by chrisspaceape** [comment]
+1 this would be amazing

**Comment by Brantone** [comment]
Sure would be sweet to be able to take in code contributions to completely some of these "easy" things.
Or even ability to run AC locally rather than hosted.

**Comment by wivlaro** [comment]
> This issue has been automatically marked as stale because it has not had any activity for 60 days. It will be closed if no further activity occurs within 15 days of this comment.

Maybe it's had no activity, but it would still be really good. Surely it's a simple feature.

**Comment by irobinSAG** [comment]
This would still be appreciated!

**Comment by Brantone** [comment]
New 60 day timer ...

**Comment by Brantone** [comment]
Bump
